README.md

```diff
@@ -168,7 +167,16 @@ Sample bands configuration file:
     "sellLimits": []
 }
 ```
-
+ ### Band examples
+Let's create some example interactions using the bands above, suppose it is denominated in Dai and the price of one Dai is $1.00 USD :
+* If we look at the first buy band, the initial buy order will be 30 Dai ( *avgAmount* ) with a price of -> `price - (price * avgMargin)` -> `1.00 -   (1.00 * 0.01)` -> $0.90 USD per Dai.
```

Comment:
Might be missing something, but where's the $0.90 come from?  `1 - (1 * 0.01) = 0.99`

Reply:
@steveklebanoff Correct, that should read `0.99`